packages/devtools_app/lib/src/screens/network/har_builder.dart

```diff
@@ -129,12 +129,20 @@ Map<String, Object?> buildHar(List<DartIOHttpRequestData> httpRequests) {
         NetworkEventKeys.dns.name: NetworkEventDefaults.dns,
         NetworkEventKeys.connect.name: NetworkEventDefaults.connect,
         NetworkEventKeys.send.name: NetworkEventDefaults.send,
-        NetworkEventKeys.wait.name: e.duration!.inMilliseconds - 2,
+        NetworkEventKeys.wait.name: e.duration?.inMilliseconds ?? 0 - 2,
```

Comment:
can you add a comment explaining the `-2`

Reply:
removing -2ms, i was trying to subtract some overhead value but this is not required.